src/auth/ejabberd_auth.erl

```diff
@@ -338,19 +343,6 @@ do_get_vh_registered_users_number(LServer, Opts) ->
             end, auth_modules(LServer))).
 
 
-%% @doc Get the password of the user.
```

Comment:
Why remove this one? Was it unused? Maybe it was intended as a debug shell utility?

Reply:
it's almost identical to `get_passterm_with_authmodule/1` interface, and we were using it only in integration tests.